File: python/shared/run_me_lib.py
def run_me(rate = 100):
    try:
        count = 100 / rate
    except Exception as e:
        count = 0
        print('some error', e)
    print('count:', count)

def print_me(user_id):
    if user_id == 1:
        print('Hello, David')
    else:
        print('Hello, John')

File: nodejs/shared/run-me-lib.js
module.exports = {
  runMe(config) {
    let count
    try {
      count = 100 / config.rate
    } catch (error) {
      count = 0
      console.log('some error', error);
    }

    console.log('count', count);
  },

  printMe(userId) {
    if (userId == 1) {
      console.log('Hello, David');
    } else {
      console.log('Hello, John');
    }
  }
}